src/WebJobs.Script/Config/MetricsOptions.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Script.Configuration
+{
+    public class MetricsOptions
```

Comment:
These values are so generic, I wouldn't call this `MetricsOptions`. Someone else that needs app name and sub could use this -- like SystemLogger, etc. You could call it something like `AppServiceOptions`? Something that says this is the options that would hold details coming from Azure...

Reply:
Renamed it